functions/go/set-image/custom/custom.go

```diff
@@ -0,0 +1,43 @@
+package custom
+
+import (
+	"fmt"
+
+	"github.com/GoogleContainerTools/kpt-functions-sdk/go/fn"
+	"sigs.k8s.io/kustomize/api/filters/imagetag"
+	"sigs.k8s.io/kustomize/api/types"
+	"sigs.k8s.io/kustomize/kyaml/filtersutil"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+)
+
+func SetAdditionalFieldSpec(image types.Image, objects fn.KubeObjects, additionalImageFields types.FsSlice, ctx *fn.Context) {
+	for i, obj := range objects {
+		objRN, err := yaml.Parse(obj.String())
+		if err != nil {
+			ctx.ResultInfo(err.Error(), obj)
```

Comment:
Is there a reason why the log is "info" but not "error"?

Reply:
Leftover after my refactor for custom, my bad. Should be error.